chips/e310x/src/chip.rs

```diff
@@ -79,9 +72,18 @@ impl<'a, I: InterruptService<()> + 'a> E310x<'a, I> {
     }
 
     pub unsafe fn enable_plic_interrupts(&self) {
-        self.plic.disable_all();
-        self.plic.clear_all_pending();
+        /* Manual, PLIC Chapter: A pending bit in the PLIC core can be cleared
```

Comment:
Please add the page and paragraph number.

Reply:
Done.